packages/edgeidentity/__tests__/IdentityMapTests.ts

```diff
@@ -53,7 +53,7 @@ describe('IdentityMap', () => {
 
     let itemsForNamespace2 = idMap.getIdentityItemsForNamespace(namespace2);
     expect(itemsForNamespace2.length).toEqual(2);
-    expect(itemsForNamespace2).toEqual([{"id": identifier3, "authenticatedState": authenticatedState3, "primary": false}, {"id": identifier4, "authenticatedState": AuthenticatedState.AMBIGUOUS, "primary": false}]);
+    expect(itemsForNamespace2).toEqual([{"id": identifier3, "authenticatedState": 'loggedOut', "primary": false}, {"id": identifier4, "authenticatedState": AuthenticatedState.AMBIGUOUS, "primary": false}]);
```

Comment:
Was this change required for identifier3?

Reply:
I want to check the test shows 'loggedOut'.
Since AuthenticatedState.LOGGED_OUT  case is similar to AuthenticatedState.AMBIGUOUS which is checked in the identifier4.